### PR TITLE
updated signal icon to the newer version

### PR DESCRIPTION
**Pull request by jorgeamadosoria**
signal
**Issue:** https://github.com/simple-icons/simple-icons/issues/6131

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -8200,7 +8200,7 @@
         },
         {
             "title": "Signal",
-            "hex": "2592E9",
+            "hex": "3977F0",
             "source": "https://play.google.com/store/apps/details?id=org.thoughtcrime.securesms"
```
Comment (service-paradis) [inline _data/simple-icons.json:8204]: The source should be https://github.com/signalapp/Signal-Desktop/blob/9db8765b6cf270195e45a7f251374d4e53d54c95/images/signal-logo.svg

**Review by service-paradis — CHANGES_REQUESTED** [review]
Hello @jorgeamadosoria' thank you for your contribution!!

I compared your SVG with https://github.com/signalapp/Signal-Desktop/blob/9db8765b6cf270195e45a7f251374d4e53d54c95/images/signal-logo.svg and it is a perfect match 💯 

If you can change the source as requested, we will be able to merge this one!

File: _data/simple-icons.json (hunk 1 above)
Comment (jorgeamadosoria) [reply _data/simple-icons.json:8204]: on it!

**Review by jorgeamadosoria — COMMENTED** [review]
updated as requested

File: _data/simple-icons.json [hunk 2]
```diff
@@ -8200,8 +8200,8 @@
         },
         {
             "title": "Signal",
-            "hex": "2592E9",
-            "source": "https://play.google.com/store/apps/details?id=org.thoughtcrime.securesms"
+            "hex": "3977F0",
```
Comment (service-paradis) [inline _data/simple-icons.json:8203]: Thank you for the changes! Just to be sure before merging, where did you take this color?
The color from the SVG source is `#3A76F0`.

Comment (jorgeamadosoria) [reply _data/simple-icons.json:8203]: from the website brand icon, here: https://signal.org/en/

Comment (jorgeamadosoria) [reply _data/simple-icons.json:8203]: I also took it from the github signal svg logo linked as source, and it gives me the same `#3977F0` value.
Where is the `#3A76F0` coming from?

I use a Chrome plugin to get colors: chrome-extension://hmdcmlfkchdmnmnmheododdhjedfccka/popup.html#
Maybe the tool is responsible for the color difference?

Comment (service-paradis) [reply _data/simple-icons.json:8239]: The color is embedded directly in the SVG:
image

The plugin might be responisible for the difference.

Comment (jorgeamadosoria) [reply _data/simple-icons.json]: That's bad news. I won't be able to trust the plugin moving forward.
Restored the hex to that color. Are we ready to merge this one?

**Review by service-paradis — APPROVED** [review]
Thank you @jorgeamadosoria, LGTM now 💯

**Pull request merged**